cores-and-flashing.tex

```diff
@@ -1,311 +1,254 @@
+\end{enumerate}
 
-  Alternatively, if you intend to flash the QSPI chip via Vivado, you would instead download the .mcs file for your target device (and optionally, the .prm files as well).
+\begin{center}
+  \fbox{\includegraphics[width=0.4\linewidth]{images/filehost_redeemmenu.png}}
+\end{center}
 
-  Another alternative for Nexys4 board users is to download .bit files and copy them to SD cards, which you can also download.
+To download the latest release package:
 
-  But once again, for the purposes of this chapter on core-flashing, you will only be interested in the .cor files.
-\fi
+\begin{enumerate}
+  \item Click the ``Files'' tab of the Filehost website.
+  \item In the search box on the left-hand side, type: ``release'' The list will update to show only files with that word in the title.
+  \item Locate the entry named, ``MEGA65 Core Release Package (mega65r3) incl. ROM,'' where ``mega65r3'' matches your hardware revision.
+  \item Click the entry. Confirm that this release package is for your hardware revision, then click ``Download'' to download the file.
+\end{enumerate}
+
+If you don't see an entry that says ``incl. ROM,'' check that you are signed in and that you have redeemed a valid owner code. Note that there is an entry for the Release Package that does not include the ROM that is visible to everyone. To ensure you are using a compatible set of files, get the package that says ``incl. ROM.''
+
+\begin{center}
+  \fbox{\includegraphics[width=0.7\linewidth]{images/filehost_release.png}}
+\end{center}
 
-\phantomsection
-\section{Selecting a Core}
+Expand the downloaded archive. You should see a file whose name ends in {\tt .cor}, and a folder of {\tt sdcard-files} that includes one named {\tt MEGA65.ROM}.
 
-To operate the MEGA65 with an alternate core, switch off the power to the MEGA65, and then hold
-\specialkey{NO SCROLL} down while switching the power back on. This instructs the MEGA65 to enter the
-Flash and Core Menu, instead of booting normally. When booting this way, the following screen will appear:
+\section{The Core Selection Menu}
+\index{Core!Core Selection Menu}
+
+The MEGA65 decides which core to load into the FPGA when it starts up. You can interrupt this process to select which core to load.\footnote{Technically, the MEGA65 starts the core in slot 0 to power the core selection menu. After you have made a selection or it chooses a default, it loads the selected core into the FPGA and continues the boot process.}
+
+To open the core selection menu, switch off the computer, then hold the \specialkey{NO\\SCROLL} key and switch on the computer. The core selection menu appears, with the eight core slots numbered 0 through 7.
 
 \begin{center}
-\includegraphics[trim= 0  0 0 10mm,clip,width=0.7\linewidth]{images/ss-flashmenu.png}
+  \includegraphics[width=0.7\linewidth]{images/ss-flashmenu.png}
```

Comment:
This would be good polish. I'm a bit exhausted from re-staging my only MEGA65 to look like a factory device, and 1st edition wasn't as careful as I've been. 😅 I'll leave this open in case I feel like deleting all my cores and taking another screenshot.

Reply:
ok done :)